AUTHORS.md

```diff
@@ -5,13 +5,14 @@ during [an NSF funded postdoc](http://www.leouieda.com/blog/hawaii-gmt-postdoc.h
 with [Paul Wessel](http://www.soest.hawaii.edu/wessel) at the University of Hawaii at
 Manoa.
 
-The following people have contributed code to the project (alphabetical by last name)
-and are considered the "PyGMT Developers":
+The following people have contributed code to the project (alphabetical by name)
+and are considered to be "PyGMT Developers":
```

Comment:
```suggestion
The following people have contributed code and/or documentation to the project
(alphabetical by name) and are considered to be "PyGMT Developers":
```
It wasn't clear to me based on AUTHORSHIP.md (which says anyone who has submitted a PR in the [``Authors.md file`` section](https://github.com/GenericMappingTools/pygmt/blob/master/AUTHORSHIP.md#the-authorsmd-file)) and this section (which says people who have contributed code) whether code is required for authorship. I'm not sure which document is accurate (this suggestion goes based on AUTHORSHIP.md).